src/khoj/interface/web/chat.html

```diff
@@ -312,6 +314,8 @@
                     imageMarkdown = `![](data:image/png;base64,${message})`;
                 } else if (intentType === "text-to-image2") {
                     imageMarkdown = `![](${message})`;
+                } else if (intentType === "text-to-image-v3") {
+                    imageMarkdown = `![](data:image/webp;base64,${message})`;
```

Comment:
It would still be using a URL, right? Is the image type specification required, or a nice to have?

Reply:
Update, the url-based `text-to-image` specification is just `text-to-image-v2`.